uitools/import/Esri/ArcGISRuntime/Toolkit/OAuth2View.qml

```diff
@@ -118,4 +118,13 @@ Dialog {
         const js = `setTimeout(function() { document.getElementById('user_username').value = '${username}'; document.getElementById('user_password').value = '${password}'; }, 1000);`;
         webView.runJavaScript(js);
     }
+
+    /*!
+      \internal
+      \brief Fill and accept the username and password
+     */
+    function applyUsernameAndPassword(username, password) {
+        const js = `setTimeout(function() { document.getElementById('user_username').value = '${username}'; document.getElementById('user_password').value = '${password}'; document.getElementById('oauth').submit(); }, 1000);`;
```

Comment:
Is it worth line-wrapping this to make it more readable? I don't think so, since we don't intend for this to be used for typical workflows.

My only thought is if it would help us troubleshoot automated test issues.

Reply:
I wasn't sure if Javascript would be happy with multi-line backtick strings, but it works fine!